src/commonMethods/commonFunctions.js

```diff
@@ -147,6 +149,7 @@ export const checkIfProcessIsRunning = function(process) {
  * @param cb
  */
 export const exec = async function(opts) {
+  deviceIP.call(this)
```

Comment:
Instead of relying on a global variable, better to do `const hostIP = deviceIP.call(this)`